pkg/fnenv/native/builtin/foreach.go

```diff
@@ -0,0 +1,82 @@
+package builtin
+
+import (
+	"fmt"
+
+	"github.com/fission/fission-workflows/pkg/types"
+	"github.com/fission/fission-workflows/pkg/types/typedvalues"
+)
+
+const (
+	Foreach            = "foreach"
+	ForeachInputHeader = "foreach"
+	ForeachInputDo     = "do"
+)
+
+// FunctionForeach is a control flow construct to execute a certain task for each element in the provided input.
+// The tasks are executed in parallel. Foreach does not gather or store the outputs of the tasks in any way.
+//
+// For example:
+// foo:
+//   run: foreach
+//   inputs:
+//     for: [],
+//     do:
+//       run: noop
+//       inputs: "{$.???}"
```

Comment:
Is the ??? a placeholder or the actual syntax?

Reply:
Yes, not done with that one yet :)